src/main/java/frc/robot/commands/DriveSlowAccel.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import static frc.robot.Constants.SwerveDrive.ROTATION_MULTIPLIER;
+import static frc.robot.Constants.SwerveDrive.SPEED_MULTIPLIER;
+
+public class DriveSlowAccel extends CommandBase {
+    private final SwerveDrive swerve;
+    private double current = 0;
+
+    public DriveSlowAccel(SwerveDrive swerve) {
+        this.swerve = swerve;
+        addRequirements(swerve);
+    }
+
+    @Override
+    public void execute() {
+        double forward = -RobotContainer.xbox.getLeftY();
+        double strafe = -RobotContainer.xbox.getLeftX();
+        double magnitude = Math.hypot(forward, strafe);
+        double alpha = Math.atan2(strafe, forward);
+        if (Math.abs(magnitude) < 0.1)
+            magnitude = 0;
+        if (magnitude == 0) current = 0;
+        magnitude *= SPEED_MULTIPLIER;
+        current += magnitude / 50;
+        if (current > magnitude) current = magnitude;
+        forward = Math.cos(alpha) * current;
+        strafe = Math.sin(alpha) * current;
+        double rotation = -RobotContainer.xbox.getRightX();
+        if (Math.abs(rotation) < 0.1) rotation = 0;
+        rotation *= ROTATION_MULTIPLIER;
+
+        if (magnitude == 0 && rotation == 0)
+            swerve.terminate();
+        else {
```

Comment:
You don't need to call the terminate method, if we drive too slow the robot just won't move.